GoogleCombined/create_list.cpp

```diff
@@ -0,0 +1,19 @@
+/// This program reads in a raw set of data points and outputs a set of data points in the correct formatting that the Swift program can read; it is specifically for the heat map feature.
```

Comment:
If you chose to add the function of generating the points, you can remove the generated data. Otherwise, you may remove the function and keep the generated points.

Reply:
I'll keep the data generation function; I'll add a note in the README so developers can either import their own data set or use this function to create the data set from raw coordinates.